beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/BlockOperationSelectorFactory.java

```diff
@@ -90,6 +90,10 @@ public Consumer<BeaconBlockBodyBuilder> createSelector(
     return bodyBuilder -> {
       final Eth1Data eth1Data = eth1DataCache.getEth1Vote(blockSlotState);
 
+      final UInt64 previousForkEpochStart =
+          spec.computePreviousForkEpochStart(blockSlotState.getSlot());
```

Comment:
I think this is actually an epoch at which the fork starts (as opposed to the slot at the start of the epoch the previous fork is configured for).  In which case I think it should be called `previousForkStartEpoch`.  Or just `firstEpochOfPreviousFork` would work too.